core/imds/src/main/java/software/amazon/awssdk/imds/internal/DefaultEc2Metadata.java

```diff
@@ -130,7 +147,64 @@ public String toString() {
      */
     @Override
     public String get(String path) {
-        return "IMDS";
+
+        String data = null;
+        AbortableInputStream abortableInputStream = null;
+        try {
+            String token = getToken();
+            URI uri = URI.create(endpoint + path);
+            HttpExecuteRequest httpExecuteRequest = requestMarshaller.createDataRequest(uri , SdkHttpMethod.GET, token,
```

Comment:
nit: extra space before comma

Reply:
Good catch . Surprising that when I ran checkstyle during local build , it did not show up as warning/error.